src/django_mysql/models/fields/fixedchar.py

```diff
@@ -26,7 +26,10 @@ def __init__(self, length: int, *args: Any, **kwargs: Any) -> None:
         if "max_length" in kwargs:
             raise TypeError('"max_length" is not a valid argument')
 
+        # However, a max_length is required by Django CharField,
+        # so we'll set it equal to the length of our CHAR field
         self.length = length
+        kwargs["max_length"] = length
```

Comment:
I considered dropping `self.length` and overloading `max_length` to store the field length and use it to generate the field in `db_type()` while keeping the `length` param but I wasn't sure if that is something you'd want.

Reply:
Yes this seems like a good idea